swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -362,26 +364,9 @@ public final class Store: ObservableObject {
     // Capture current hash before IPC call
     let currentHash = resourceListHash
 
-    // Get data from the provider - if hash matches, provider returns nil
-    let data = try await withCheckedThrowingContinuation {
-      (continuation: CheckedContinuation<Data?, Error>) in
-      do {
-        guard session.status == .connected else {
-          throw IPCClient.Error.invalidStatus(session.status)
-        }
-
-        try session.sendProviderMessage(
-          IPCClient.encoder.encode(ProviderMessage.getResourceList(currentHash))
-        ) { data in
-          continuation.resume(returning: data)
-        }
-      } catch {
-        continuation.resume(throwing: error)
-      }
-    }
-
```

Comment:
This business logic was moved back into IPCClient, where we can leverage the new `sendProviderMessage` private func to have consistent IPC call semantics.

Not sure how it escaped, but it would probably be a tad cleaner to keep IPC calls in that module instead of calling them directly from other areas of the codebase.